src/Player.js

```diff
@@ -1,21 +1,64 @@
-let EventEmitter = require("events").EventEmitter
-
-/**
- * Represents a Player system
- * @extends EventEmitter
- * @prop {VoiceChannel} channel The voice channel the player is playing to
- * @prop {Guild} guild The guild the voice channel is in
- * @prop {Node} node The node the player is broadcasting on
- * @prop {Boolean} ready Whether or not the node is connected to the voice channel
- * @prop {Object} nowPlaying The currently playing track's info
- */
+const EventEmitter = require("events").EventEmitter
+
 module.exports = class Player extends EventEmitter {
+  /**
+   * Represents a Player system
+   * @extends EventEmitter
+   * @prop {VoiceChannel} channel The voice channel the player is playing to
```

Comment:
You forgot to remove this from the top after you moved it into the constructor body